### ISSUE TITLE
[v6-rc4] Form-wide error object not defined even when using _error key on synch validate

### ISSUE THREAD

**Issue by JMacLulich** [erikras/redux-form#1533]
Hi Guys,

I've been going over and over the documentation and source trying to find what I'm doing wrong with this, but I have't had any luck.

I'm trying to return a `_error` object from my synchronous validation function so I can access it as the `error` property during my form render.

The documentation here: http://redux-form.com/6.0.0-rc.4/docs/api/Props.md/ (specifically for the item `error`) says I should assign a value to the key `_error`. I'm also following the example for submit validation here: http://redux-form.com/6.0.0-rc.4/examples/submitValidation/ which shows the use of the `_error` field when throwing a `SubmissionError `.

Except, I continue to get `undefined` as the value of `error`.

For example:

```javascript

function synchValidate(values) { 
  
   const errors = { _error: {} }
   errors._error = "This is an error" 

   return errors
}

function render() { 

  const { error } = this.props
  
  console.log(error)
}
```

I'm expecting that the string "This is an error" is output when the render function of the form is called but I'm not getting it.

I'm probably missing something obvious, any insights would be great.

If you need a more detailed example I'm happy to produce  

Thanks, Jason.

**Comment by JMacLulich** [comment]
Just as an additional point.

I can see that the `syncValidate` field is in the store under the right form.

![image](https://cloud.githubusercontent.com/assets/1563391/17666056/c35ad250-6341-11e6-9335-9d45b9d736c1.png)

you can see here, that the `syncErrors` field as the field `_error` which I thought would be accessible as a property on the form inside the `render` method.

But it's not, I really need access to that `_error` field so I can figure out errors that are not associated with a specific field, but rather behaviour coordinated between a range of fields.

**Comment by bmv437** [comment]
I'm experiencing the exact same issue as well.

**Comment by erikras** [comment]
Fixed by #1560. I'll comment here again when it has been released.

**Comment by JMacLulich** [comment]
@erikras great thanks for that! looking forward to rc5

**Comment by erikras** [comment]
Fix published in [`v6.0.0-rc.5`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-rc.5).
